source/effect_preprocessor.hpp

```diff
@@ -85,7 +92,7 @@ namespace reshadefx
 		/// Get a list of all defines that were used in #ifdef and #ifndef lines
 		/// </summary>
 		/// <returns></returns>
-		std::vector<std::pair<std::string, std::string>> used_macro_definitions() const;
+		std::vector<std::pair<std::string, macro_detection_info>> used_macro_definitions() const;
```

Comment:
It looks like the additional information returned here is not actually used anywhere currently, so this change is probably not necessary (yet).